tests/integration/reqs/test_submit_multisigned.py

```diff
@@ -1,115 +1,59 @@
-        signer_entries=[
-            SignerEntry(
-                account=FIRST_SIGNER.classic_address,
-                signer_weight=1,
-            ),
-            SignerEntry(
-                account=SECOND_SIGNER.classic_address,
-                signer_weight=1,
-            ),
-        ],
+        signer_entries=SIGNER_ENTRIES,
     ),
     WALLET,
 )
 
 
 class TestSubmitMultisigned(IntegrationTestCase):
-    @test_async_and_sync(globals())
+    @test_async_and_sync(
+        globals(),
+        [
+            "xrpl.transaction.sign",
+            "xrpl.transaction.autofill",
+        ],
+    )
     async def test_basic_functionality(self, client):
-        #
-        # Perform multisign
-        #
-        # NOTE: If you need to use xrpl-py for multisigning, please create an issue on
-        # the repo. We'd like to gauge interest in higher level multisigning
-        # functionality.
         issuer = Wallet.create()
         tx = TrustSet(
             account=WALLET.classic_address,
-            sequence=WALLET.sequence,
-            fee=FEE,
-            flags=TrustSetFlag.TF_SET_NO_RIPPLE,
-            limit_amount=IssuedCurrencyAmount(
-                issuer=issuer.classic_address,
-                currency="USD",
-                value="10",
-            ),
-        )
-        tx_json = tx.to_xrpl()
-        first_sig = sign(
-            bytes.fromhex(
-                encode_for_multisigning(
-                    tx_json,
-                    FIRST_SIGNER.classic_address,
-                )
-            ),
-            FIRST_SIGNER.private_key,
-        )
-        second_sig = sign(
-            bytes.fromhex(
-                encode_for_multisigning(
-                    tx_json,
-                    SECOND_SIGNER.classic_address,
-                )
-            ),
-            SECOND_SIGNER.private_key,
-        )
-        multisigned_tx = TrustSet(
-            account=WALLET.classic_address,
-            sequence=WALLET.sequence,
-            fee=FEE,
             flags=TrustSetFlag.TF_SET_NO_RIPPLE,
             limit_amount=IssuedCurrencyAmount(
                 issuer=issuer.classic_address,
                 currency="USD",
                 value="10",
             ),
-            signers=[
-                Signer(
-                    account=FIRST_SIGNER.classic_address,
-                    txn_signature=first_sig,
-                    signing_pub_key=FIRST_SIGNER.public_key,
-                ),
-                Signer(
-                    account=SECOND_SIGNER.classic_address,
-                    txn_signature=second_sig,
-                    signing_pub_key=SECOND_SIGNER.public_key,
-                ),
-            ],
         )
+        autofilled_tx = await autofill(tx, client, len(SIGNER_ENTRIES))
+        tx_1 = await sign(autofilled_tx, FIRST_SIGNER, multisign=True)
+        tx_2 = await sign(autofilled_tx, SECOND_SIGNER, multisign=True)
+        multisigned_tx = multisign(autofilled_tx, [tx_1, tx_2])
```

Comment:
Two recommendations:
1. I'd recommend using a fixed FIRST_SIGNER / SECOND_SIGNER, and then doing a check that `multisign(autofilled_tx, [tx_1, tx_2])` matches a pre-computed value of what that should be. (You can use `xrpl.js`'s multisign for those fixed signer values as a reference to see what the correct output for `multisign` should be) - The reason for this is it acts as a unit test going forward ensuring that `multisign` gives an accurate value for a given wallet - not just one that rippled receives successfully. 

2.  Since this is the defacto integration test for multisign, I think it should go through the proper steps of setting up signers and checking that rippled actually allows the multisigned transaction to take effect. (Basically doing the same steps you have in `multisign.py` but replacing the print statements with asserts :) - Otherwise I'm worried that the test could pass while multisigned transactions are being rejected by rippled.